src/window.rs

```diff
@@ -15,6 +15,8 @@ use crate::{
 
 pub use crate::icon::{BadIcon, Icon};
 
+pub type CursorIcon = cursor_icon::CursorIcon;
```

Comment:
This type alias should be documented.

Actually, it might be better to just inline the whole `cursor_icon` crate into the `window` module, like so:

```rust
#[doc(inline)]
pub use cursor_icon::{CursorIcon, ParseError as CursorIconParseError};
```